frontend/src/api.ts

```diff
@@ -80,3 +80,11 @@ interface MeResponse {
 export function me(): Promise<MeResponse> {
   return apiCall<MeResponse>('/api/me');
 }
+
+type OrgsResponse = string[];
```

Comment:
let's use correct type from the beginning. We will need id or type or image or something else anyway. Let's return it as organization object already. (with only field name)

Reply:
If I understood you correctly, done in 48c9763

Reply:
yes. That's what I meant. Thanks!